### DISCUSSION TITLE
Should I expect it works using a facebook test phone number?

### DISCUSSION THREAD

**Discussion by bvicini** [netflie/whatsapp-cloud-api#33] [General]
I created Meta account with test phone number. Messages to one of the five usable phone number work from facebook app, but not from this library:  no error is fired and the server is on LAN, for more complete information. Is the problem about using a test account? Or maybe relates on working on LAN? Thnak you in advance for the answer and your great work.

**Comment by aalbarca** [comment]
Hi @bvicini thanks for your support :)

Please, check this topic https://github.com/netflie/whatsapp-cloud-api/issues/5

Best regards,
Alex.

**Reply by bvicini** [reply]
Thank you, i did not replay to the frist message. Sorry, didn't find this post. Great job
